src/odemis/acq/test/acq_test.py

```diff
@@ -339,7 +339,8 @@ def test_metadata(self):
         specs2.detExposureTime.value = 0.05
 
         f = acqmng.acquire([sps, sps2], settings_obs)
-        data = f.result()
+        data, exp = f.result()
+        self.assertIsNone(exp)
```

Comment:
I think same for line 236,  self.assertIsNone(exp) is missing

Reply:
Good catch! I'll add the check there too.